manifests/bind_event.pp

```diff
@@ -0,0 +1,27 @@
+# Define: qpid::bind_event
+#
+# This define binds the event queue with the correct messages
+#
+# === Parameters
+#
+# $hostname::                   Set to localhost for qpid-config operations
+#
+# $ssl_cert::                    SSL cert to use for qpid-config commands
+#
+# $queue::                      Name of the event queue
+#
+# $port::                       Port that qpid is listening on
+define qpid::bind_event(
+  $hostname = 'localhost',
+  $ssl_cert = undef,
+  $queue = 'event_queue',
+  $port = '5671'
+) {
+    exec { "bind entitlements queue to qpid exchange messages that deal with ${title}":
+    command   => "qpid-config --ssl-certificate ${ssl_cert} -b 'amqps://${hostname}:${port}' bind event ${queue} ${title}",
+    onlyif    => "qpid-config --ssl-certificate ${ssl_cert} -b 'amqps://${hostname}:${port}' exchanges event -r | grep ${title}",
```

Comment:
One improvement we could make here is supporting non-ssl connections.  It currently assumes that it's always SSL. You can do something like this above the exec

```puppet
if($ssl_cert) {
  $ssl_option = "--ssl-certificate ${ssl_cert}"
  $protocol = "ampqs"
} else {
  $ssl_option = ""
  $protocol = "amqp"
}
```

and then put `${protocol}` and `${ssl_option}` in the right places in command